array/zipN.js

```diff
@@ -1,2 +1,7 @@
-export default (xs, ...ys) =>
-  xs.map((value, index) => ys.reduce((x, xs) => [...x, xs[index]], [value]));
+export default (...xs) => {
+  const [head, ...tail] = xs;
+
+  return head.map((value, index) =>
```

Comment:
This will fail for the case `zipN()` - please guard against it and cover it with tests.